lms/static/sass/_build-lms.scss

```diff
@@ -52,6 +52,7 @@
 @import 'views/decoupled-verification';
 @import 'views/shoppingcart';
 @import 'views/homepage';
+@import 'views/creative-commons';
```

Comment:
Yeah, sadly our LMS sass compile/partial organization has lots of barnacles. Your best best is to place the ``creative-commons`` sass partial in https://github.com/edx/edx-platform/tree/sass-output-to-css-dir/lms/static/sass/elements. Sound good?